### ISSUE TITLE
[Docs] [Docs I/A] Apply new home page design

### ISSUE THREAD

**Issue by roadlittledawn** [newrelic/docs-website#4431]
## Summary

Apply design to `src/pages/index.js`.

I don't think any of this _needs_ to be in the theme.

For any design related questions, feel free to reach out to Àngel Ordax

### Questions

- [x]  icons: for time being, can we stick with feather instead of switching to eva based icons?

> `note` this ticket will not introduce a new icon library

### Notes on design
- `Popular content` section: Hard coded list of links. Tech docs team will update these periodically.
- `What's new` section: 3 most recent posts. Use `frontmatter.title` as secondary sorting criteria when multiple posts exist with same datestamp in `frontmatter.releaseDate`
- Tiles: Note the hover effect in the figma mock (on Data explorer)
- ~~Icons: Uses library called [Eva](https://akveo.github.io/eva-icons/#/). Uses SVG similar to feather.~~
- Links: Should use current teal on light/dark (not all design mocks may have been updated for that)

**Figma**
[Desktop (light mode)](https://www.figma.com/proto/0jsTPLtwz6Bljndkbr1COn/open-source-DS-%5BWIP%5D?page-id=56%3A39509&node-id=150%3A52221&viewport=501%2C48%2C1.05&scaling=min-zoom&starting-point-node-id=150%3A52221&show-proto-sidebar=1)

![2021-11-09_09-16-22](https://user-images.githubusercontent.com/2952843/140972707-9052cb61-4a8e-461d-b906-b84a652912c1.png)

[Desktop (light mode)](https://www.figma.com/proto/0jsTPLtwz6Bljndkbr1COn/open-source-DS-%5BWIP%5D?page-id=56%3A39509&node-id=171%3A40520&viewport=501%2C48%2C1.05&scaling=min-zoom&starting-point-node-id=171%3A40520&show-proto-sidebar=1)

[Mobile](https://www.figma.com/proto/0jsTPLtwz6Bljndkbr1COn/open-source-DS-%5BWIP%5D?page-id=141%3A43127&node-id=141%3A43617&viewport=501%2C48%2C1.01&scaling=scale-down&starting-point-node-id=141%3A43617)

## Acceptance criteria

- [ ] Home page updated per design mock spec
- [ ] Buttons in banner are instrumented via Tessen with an attribute/property that identifies it came from home page banner

## Helpful links

**Comment by jpvajda** [comment]
@roadlittledawn thought by end of this week, this design may be finalized.

**Comment by jpvajda** [comment]
Final design is not 💯 yet, but it's pretty close and @roadlittledawn was going to break this ticket down with what is approved so we can get started.

**Comment by jpvajda** [comment]
@legnaleama  What is the rational for using a different icon library at this stage of this improvement?  Can we just use what is already available to our sites from new relic and feather.

`Icons: Uses library called Eva. Uses SVG similar to feather`

**Comment by roadlittledawn** [comment]
@legnaleama said he'll take a look tomorrow about using feather icon equivalents for now. and any icons without match we can either use something from feather OR use SVG value

**Comment by roadlittledawn** [comment]
Worth noting that we have some components that handle feather icon SVG in a way that could technically work with Eva. So "switching libraries" could entail tweaking those components (`FeatherSVG` and `SVG`) a little bit or creating separate ones. i think there's probably some related functionality in the `Icon` component as well.

**Comment by jpvajda** [comment]
@roadlittledawn I totally think it be worth exploring that effort to switch libraries in the future, did you want to make a follow up 🎫 for that? If we have similar feather / svg icons to use that are already available my thought is we just de-scope this until later given our current roadmap demands for Q3.

**Comment by roadlittledawn** [comment]
@jpvajda i'm split i guess. our "library" is just some components around SVGs really. there's only a subset of feather icons available and if you want a new one you have to add the SVG value manually.

i think we could probably tackle this as part of this sprint if we really wanted to (even if it's just copying what we do for feather). either way, it's probably a separate ticket so i can file a ticket for that.

**Comment by legnaleama** [comment]
@jpvajda @roadlittledawn 
re: Feather icons. I was unaware of the usage of that library at the time of creating the mockup. Eva is just a library I've used in the past as a good base for defining an icon library. That said, I checked Feather and started implementing it on the DS and even though some icons had rendering issues I had to correct, for the most part I'm okay leaving it as it is for now. 

As a side project, I'd be interested to know what's the most efficient way to handle SVG icons and I'm already discussing ways to do it from the Design (Figma) side of things with OneCore to support external Design Systems. 

**TLTR: We'll keep the feather icons for now.**
